.travis.d/templates/configmap.yaml.erb

```diff
@@ -1,10 +1,11 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: <%= "#{key}-#{file['owner']}" %>
+  name: <%= "#{app_name}" %>
   labels:
-    name: <%= "#{key}-#{file['owner']}" %>
+    name: <%= "#{app_name}" %>
 data:
-  path: <%= file['path'] %>
-  contents: '<%= file['contents'].gsub("\n", Regexp.escape("\\n")) %>'
-  
\ No newline at end of file
+  <% app['files']&.each do |key, file| %>
+  <%= file['path'] %>: |-
+    <%= file['contents'].gsub("\n", "\n    ") %>
```

Comment:
`\n` might not be compatible with all line endings, maybe try `/^/\t/` to handle both line endings?

Reply:
can you clarify this?
sorry :(